dotcom-rendering/src/components/SubNav.importable.tsx

```diff
@@ -116,28 +122,51 @@ const showMoreStyle = css`
 	}
 `;
 
-const listItemStyles = css`
-	:after {
-		content: '';
-		display: inline-block;
-		width: 0;
-		height: 0;
-		border-top: 6px solid transparent;
-		border-bottom: 6px solid transparent;
-		border-left: 10px solid ${palette('--sub-nav-border')};
-		margin-top: 12px;
-		margin-left: 2px;
+const frontSubNavBorder = (currentPillarTitle: string) => {
+	switch (currentPillarTitle.toLowerCase()) {
+		case 'news':
+			return sourcePalette.news[400];
+		case 'opinion':
+			return sourcePalette.opinion[400];
+		case 'sport':
+			return sourcePalette.sport[400];
+		case 'culture':
+			return sourcePalette.culture[400];
+		case 'lifestyle':
+			return sourcePalette.lifestyle[400];
+		default:
+			return sourcePalette.news[400];
+	}
+};
 
-		${from.tablet} {
-			margin-top: 16px;
+const listItemStyle = (currentPillarTitle?: string) => {
+	const borderColor = currentPillarTitle
+		? frontSubNavBorder(currentPillarTitle)
+		: palette('--sub-nav-border');
```

Comment:
I wonder if we can simplify this by moving frontSubNavBorder in to palette and switching between the two palette options with an `isCurrentPillar` boolean? This would keep the pillar choices in the palette file which is in keeping with DCR design